Concentration/ViewController.swift

```diff
@@ -77,6 +77,19 @@ class ViewController: UIViewController {
         }
     }
     
+    /* A dictionary mapping themes (Strings) to tuples of the form (array, color1, color2) where
+     "array" is an array of emoji choices, "color1" is a color literal for the background (bg) of the cards,
+     and "color2" is a color literal for the overall bg.
+    */
+    let themesDict = ["Halloween": (["🦇", "😱", "🙀", "😈", "🎃", "👻", "🍭", "🍬", "🍎"], #colorLiteral(red: 1, green: 0.5781051517, blue: 0, alpha: 1), #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)),
+                      "Animals": (["🙀", "🦇", "🐱", "🐥", "🐦", "🦉", "🐺", "🦄", "🐷" ], #colorLiteral(red: 0.4666666687, green: 0.7647058964, blue: 0.2666666806, alpha: 1), #colorLiteral(red: 0.7254902124, green: 0.4784313738, blue: 0.09803921729, alpha: 1)),
+                      "Sports": (["⚽️", "🥎", "⚾️", "🥋", "🎽", "🥏", "🤿","🏒", "🏉" ], #colorLiteral(red: 0.2196078449, green: 0.007843137719, blue: 0.8549019694, alpha: 1), #colorLiteral(red: 0.9764705896, green: 0.850980401, blue: 0.5490196347, alpha: 1)),
+                      "Food": (["🍭", "🎃", "🍎", "🍬", "🍌", "🥝" ,"🍞", "🥕", "🥯"], #colorLiteral(red: 0.8549019694, green: 0.250980407, blue: 0.4784313738, alpha: 1), #colorLiteral(red: 0.1764705926, green: 0.4980392158, blue: 0.7568627596, alpha: 1)),
+                      "Faces": (["😱", "😀", "😋", "🥸", "😜", "🥳", "😛", "🤩", "😏"], #colorLiteral(red: 0.6000000238, green: 0.6000000238, blue: 0.6000000238, alpha: 1), #colorLiteral(red: 0.8039215803, green: 0.8039215803, blue: 0.8039215803, alpha: 1)),
+                      "Flags": (["🏳️‍🌈", "🏳️", "🏴","🏴‍☠️","🏁","🇧🇿","🇫🇯","🇹🇭","🇮🇳"], #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1), #colorLiteral(red: 0.2588235438, green: 0.7568627596, blue: 0.9686274529, alpha: 1))
+                    ]
```

Comment:
You define a dictionary from string to tuple. You can define an array of a theme object like:
```
[
[name: "Halloween", color: ..., emojies: ...],
[name: "Animals", color: ..., emojies: ...],
]
...